bin/tasks.js

```diff
@@ -4,12 +4,11 @@ const path = require('path');
 const cwd = process.cwd();
 const target = path.resolve(cwd, 'public');
 const source = path.resolve(__dirname, '../src');
-const sass = require('witch')('node-sass');
 
 // jscs:disable maximumLineLength
 module.exports = {
   'make-folders': `mkdir -p ${target}/js ${target}/css ${target}/images`,
-  'compile-css': `${sass} ${source}/scss/app.scss ${target}/css/app.css --include-path ./node_modules`,
+  'compile-css': `npmsass ${source}/scss/app.scss > ${target}/css/app.css`,
```

Comment:
Actually, there's an outside chance you *might* not need to, npm might work it out since you're inside the `bin` from package.json, but I'd want to check that first.

`witch` will be safe though.